src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -373,7 +373,10 @@ const TwoWindingsTransformerCreationDialog = ({
         validation: {
             isFieldRequired: ratioTapChangerEnabled,
             valueGreaterThan: ratioLowTapPosition - 1,
-            valueLessThanOrEqualTo: ratioHighTapPosition,
+            valueLessThanOrEqualTo:
+                editData || isCopy
```

Comment:
I think ratioHighTapPosition should be used in priority if it is defined. Don't you ?

Reply:
I agree, will change this